sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/LexicalTokenizerNameConverter.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.implementation.converters;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.search.documents.models.LexicalTokenizerName;
+
+/**
+ * A converter between {@link com.azure.search.documents.implementation.models.LexicalTokenizerName} and
+ * {@link LexicalTokenizerName}.
+ */
+public final class LexicalTokenizerNameConverter {
+    private static final ClientLogger LOGGER = new ClientLogger(LexicalTokenizerNameConverter.class);
+
+    /**
+     * Maps from enum {@link com.azure.search.documents.implementation.models.LexicalTokenizerName} to enum
+     * {@link LexicalTokenizerName}.
+     */
+    public static LexicalTokenizerName map(com.azure.search.documents.implementation.models.LexicalTokenizerName obj) {
+        if (obj == null) {
+            return null;
+        }
+        return LexicalTokenizerName.fromString(obj.toString());
```

Comment:
Given that this is an enum does it really need a custom and implementation representation?